python_modules/dagster/dagster/core/asset_defs/assets_from_modules.py

```diff
@@ -71,7 +71,10 @@ def assets_and_source_assets_from_modules(
 
 
 def assets_from_modules(
-    modules: Iterable[ModuleType], extra_source_assets: Optional[Sequence[SourceAsset]] = None
+    modules: Iterable[ModuleType],
+    group_name: Optional[str] = None,
+    asset_key_prefix: Optional[str] = None,
+    extra_source_assets: Optional[Sequence[SourceAsset]] = None,
```

Comment:
I should have caught this the first time around: now that this just returns a list, I don't think there's any need for this `extra_source_assets` argument. The user could just concatenate this list to the list afterwards.

Reply:
Got it--removed these.